far/FarRus.hlf.m4

```diff
@@ -6063,109 +6063,157 @@ $ #far:config Interface.RedrawTimeout#
+ Этот числовой параметр определяет максимальный размер данных файла,
+используемых, чтобы найти плагин, поддерживающий этот файл.
+
+ Когда пользователь нажимает #Enter# или #Ctrl+PgDn#, Far Manager читает
+указанное количество байт от начала файла и передаёт эти данные
+зарегистрированным плагинам проверить, не могут ли они «открыть» или
+иным способом отобразить файл.
 
  Минимальное значение - 0x1000. Максимальное - 0xFFFFFFFF.
 
- Не рекомендуется выставлять значение этого параметра больше 5 Mb.
+ Не рекомендуется задавать значение этого параметра больше 5 MB.
 
- По умолчанию значение = 0x20000.
+ Значение по умолчанию: 0x20000.
 
  Изменить этот параметр можно только через ~far:config~@FarConfig@.
 
 
 @System.SetAttrFolderRules
 $ #far:config System.SetAttrFolderRules#
- Параметр "System.SetAttrFolderRules" позволяет задавать значение по умолчанию опции "Process subfolders"
-в диалоге установки атрибутов для одиночной папки:
+ Этот логический (Boolean) параметр определяет значение по умолчанию
+опции #Обрабатывать вложенные папки# диалога ~Атрибутов~@FileAttrDlg@
+файлов при изменении атрибутов одной папки.
 
- true  - ^<wrap>опция "Process subfolders" выключена, файловые дата и время установлены.
- false - ^<wrap>опция "Process subfolders" включена, файловые дата и время очищены.
+ False - ^<wrap>Опция #Обрабатывать вложенные папки# включена; поля даты
+и времени очищены.
+ True  - Опция #Обрабатывать вложенные папки# выключена; поля даты
+и времени заполнены актуальными значениями.
 
- По умолчанию значение = true.
+ Значение по умолчанию: True (не обрабатывать вложенные папки).
 
  Изменить этот параметр можно только через ~far:config~@FarConfig@.
 
 
 @System.CopyBufferSize
 $ #far:config System.CopyBufferSize#
- Параметр "System.CopyBufferSize" задаёт размер буфера, когда не используется
-~системная функция копирования~@SystemSettings@. Если параметр равен 0, то используется
-размер по умолчанию - 32768 байт.
+ Этот числовой параметр определяет размер буфера, используемого
+внутренней процедурой копирования файлов (см. опцию
+#Использовать системную функцию копирования# диалога настройки
+~Системных параметров~@SystemSettings@).
+
+ Если значение этого параметра равно нулю, то используется размер буфера
+по умолчанию, 32768 байт.
 
- По умолчанию значение равно 0.
+ Значение по умолчанию: 0 (размер буфера 32768 байт).
 
  Изменить этот параметр можно только через ~far:config~@FarConfig@.
 
 
 @System.SubstNameRule
 $ #far:config System.SubstNameRule#
- Параметр "System.SubstNameRule" задаёт правило опроса приводов на предмет сканирования SUBST-дисков.
+ Этот числовой параметр задаёт типы физических носителей, которые будут
+опрашиваться, когда Far Manager определяет, какие диски были назначены
+командой #SUBST#.
 
- Биты:
- 0 - если установлен, то опрашивать сменные диски
- 1 - если установлен, то опрашивать все остальные
+ Far Manager пытается определить, является ли диск «назначенным», чтобы
```

Comment:
> «назначенным»

Microsoft [translates](https://www.microsoft.com/en-us/language/Search?&searchTerm=subst&langID=635&Source=true&productid=526) 'substituted' as either 'отображенный' or 'виртуальный'.

Reply:
I meant the translation used in this snippet:
```
Windows cannot check a disk that is substituted or
assigned using the SUBST or ASSIGN command.
```
```
Windows не может проверить диск, который отображен
или назначен с помощью команды SUBST или ASSIGN.
```
Sounds good to my ear.

Now, «назначенный диск» for “substituted disk” is a bit questionable, but I think mostly because it is somewhat colloquial. On the other hand, why not?

I requested opinion of two native Russian speakers. Both are perfectly OK with my choice; one of them even insisted that the quotes around «назначенный» must be omitted (which I tend to agree with).

The word 'виртуальный' is overloaded and would have to be introduced with more prose. And I am sorry, I won't even comment on 'отображенный'. :) The author of the original article used `SUBST-диск`, which is also awkward, but maybe?

I'd rather keep my wording. If you disagree, please help me with this one; it happens to be tough.